cumulusci.yml

```diff
@@ -229,6 +229,11 @@ tasks:
             path: scripts/setup.cls
             apex: qaSetup();
 
+    github_release_notes:
+        description: Creates a section in github release notes for a trialforce template ID placeholder.
```

Comment:
I'd suggest not adding the description here as it'll override the description for the whole release notes task.